morse.rb

```diff
@@ -0,0 +1,52 @@
+MORSE_CODE = {
+  '.-' => 'A', '-...' => 'B', '-.-.' => 'C',
+  '-..' => 'D', '.'    => 'E', '..-.' => 'F',
+  '--.' => 'G', '....' => 'H', '..'   => 'I',
+  '.---' => 'J', '-.-' => 'K', '.-..' => 'L',
+  '--' => 'M', '-.' => 'N', '---' => 'O',
+  '.--.' => 'P', '--.-' => 'Q', '.-.' => 'R',
+  '...' => 'S', '-' => 'T', '..-' => 'U',
+  '...-' => 'V', '.--'  => 'W', '-..-' => 'X',
+  '-.--' => 'Y', '--..' => 'Z'
+}.freeze
+
+def decode_char(char)
+  puts MORSE_CODE[char].upcase
+end
```

Comment:
Considering you'll be using this method in the `decode_word` method, you should make it return the character since `puts` returns the value `nil`.  You can either remove `puts` or replace it with the keyword `return`.